hack/make-rules/verify.sh

```diff
@@ -178,6 +182,16 @@ function run-checks {
   done
 }
 
+function target-checks {
```

Comment:
Can we name this something like `missing-target-checks` instead so it's clear what it's doing. A comment wouldn't be a bad idea either.

Reply:
I will address it in the latest diff.